### PR TITLE
Change Bitergia Analytics plugin zip url

### LINKED ISSUE #2: Install official Bitergia Analytics Plugin
The [current version of the Dockerfile](https://github.com/Bitergia/bitergia-analytics-odfe-kibana/blob/master/Dockerfile#L7) doesn't install the official version of the Bitergia Analytics Plugin. It takes it from the old repository.

It should point to: https://github.com/Bitergia/bitergia-analytics-plugin/releases/download/0.0.1_7.10.0/bitergiaAnalytics-7.10.0.zip

### PR DESCRIPTION
**Pull request by dlumbrer**
This PR should fix #2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ WORKDIR /usr/share/kibana
 
 ENV PATH=/usr/share/kibana/bin:$PATH
 
-RUN kibana-plugin install "https://github.com/dlumbrer/bitergia-analytics-plugin/releases/download/0.0.1_7.10.1/bitergiaAnalytics-7.10.0.zip"
+RUN kibana-plugin install "https://github.com/Bitergia/bitergia-analytics-plugin/releases/download/0.0.1_7.10.0/bitergiaAnalytics-7.10.0.zip"
```
Comment (sduenas) [inline Dockerfile:7]: The minor number of the version is `1` and `0` in the other. Is there any specific reason for it?

**Review by sduenas — CHANGES_REQUESTED** [review]
Looks good to me but we need to clarify the comment about the version number.

File: Dockerfile (hunk 1 above)
Comment (dlumbrer) [reply Dockerfile:7]: That is for the Kibana version used for the plugin (in this case is 7.10.0 for Open Distro 1.12.0), I'm my personal repository release I put `7.10.1` by mistake, the right version number is `7.10.0`

**Comment by dlumbrer** [comment]
> Looks good to me but we need to clarify the comment about the version number.

I clarified it in the conversation about the change.

File: Dockerfile (hunk 1 above)
Comment (sduenas) [reply Dockerfile:7]: Understood. No changes are needed.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**